graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/directives/LinkImportsRenamingVisitor.java

```diff
@@ -1,7 +1,8 @@
-package com.apollographql.federation.graphqljava;
+package com.apollographql.federation.graphqljava.directives;
```

Comment:
renaming visitor has a default class visibility so it is not part of the public API and can be moved to different package without causing a breaking change